config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -262,7 +285,7 @@ private Filter buildConfigResourceContextsFilter(
             .map(this::getConfigResourceFieldContextFilter)
             .collect(Collectors.toUnmodifiableList());
     Filter configResourceFieldContextFilter = new Filter();
-    configResourceFieldContextFilter.setOp(Filter.Op.OR);
+    configResourceFieldContextFilter.setOp(OR);
```

Comment:
In this PR or another - it'd be good to move this filter building into the filter builder class now that there's a dedicated class for it.

Reply:
ok making those refactor in a different PR